internal/app/singularity/remote_get_login_password.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2018-2023, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+package singularity
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	scslibclient "github.com/sylabs/scs-library-client/client"
+	"github.com/sylabs/singularity/pkg/sylog"
+)
+
+var errNotSupported = errors.New("This library does not support direct OCI access")
+
+// RemoteGetLoginPassword retrieves cli token from oci library shim
+func RemoteGetLoginPassword(config *scslibclient.Config) (string, error) {
+	client := http.DefaultClient
+	path := "/v1/rbac/users/current"
+	endPoint := config.BaseURL + path
+
+	req, err := http.NewRequest(http.MethodGet, endPoint, nil)
+	if err != nil {
+		return "", errNotSupported
+	}
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", config.AuthToken))
+	res, err := client.Do(req)
+	sylog.Debugf("res: %v", res)
```

Comment:
If Debug code is left it, it should be more descriptive.

I would be inclined to remove this one, as we can trace the request by using `GODEBUG=http2debug=2` or similar.... and leaving it in dumps an auth token in `--debug` output that people commonly post publicly.

Reply:
removed